src/components/graphics/TwilioLogo.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { TWILIO_ASCII } from '../../utils/asciiLogos';
+import { TWILIO_LARGE } from '../../utils/asciiLogos';
```

Comment:
Wait, there was already a Twilio Logo in the source for us to use? 😂 Here I thought we'd need to make one from scratch

Reply:
A very welcome surprise indeed 😁